scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedJvmTaskExecutionCommandCreator.java

```diff
@@ -96,9 +97,11 @@
             jvmArguments.add(CentralPAPropertyRepository.PA_CONFIGURATION_FILE.getCmdLine() +
                              CentralPAPropertyRepository.PA_CONFIGURATION_FILE.getValue());
         }
+
         if (CentralPAPropertyRepository.JAVA_SECURITY_POLICY.isSet()) {
-            jvmArguments.add(CentralPAPropertyRepository.JAVA_SECURITY_POLICY.getCmdLine() +
-                             CentralPAPropertyRepository.JAVA_SECURITY_POLICY.getValue());
+            jvmArguments.add("-Djava.security.policy=" +
```

Comment:
please use CentralPAPropertyRepository.JAVA_SECURITY_POLICY.getCmdLine() +

Reply:
Ok, it's done.